sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Release History
 
 ## 1.4.0b4 (Unreleased)
+- `azure.identity.aio.AuthorizationCodeCredential.get_token()` no longer accepts
+  optional keyword arguments `executor` or `loop`. Prior versions of the method
```

Comment:
`executor` & `loop` were already in 1.3.1?

Reply:
Yes, they've been around since 1.0.0b4.

Reply:
So do we really want 1.4.0 to break 1.3.1?

Reply:
As I wrote at the top of this PR, these arguments never worked. Trying to use them just raises exceptions.

Reply:
So the behavior is 
Raise exception -> silently ignored?

We should add it into Breaking Change section